runners/console/index.ts

```diff
@@ -742,4 +743,36 @@ export class Console extends Runner {
         }
     }
 
+    runExecuteCommand(runCommand: RunCommand): RunResult {
+        let result = new RunResult();
+        result.returnCode = 0;
+        let exeCommandWithPath = runCommand.command.parameters[0];
+        
+        if(this.platform == ConsolePlatform.WINDOWS)
+        {
+            exeCommandWithPath = "powershell.exe -command "+ exeCommandWithPath;
+            if(exeCommandWithPath.includes("bash "))
+            {
+                exeCommandWithPath =exeCommandWithPath.replace("bash ", ".\\");
+            }
+        }
+
+        if(runCommand.command.parameters.length > 1 && runCommand.command.parameters[1].args) 
+            exeCommandWithPath = exeCommandWithPath+ " " +runCommand.command.parameters[1].args.join(" ");
+
+        let dirPath = runCommand.command.parameters.length > 1 && runCommand.command.parameters[1].dir
+        ? path.join(this.getWorkingDirectory(), runCommand.command.parameters[1].dir)
+        : this.getWorkingDirectory();
+
+        if(runCommand.command.parameters.length > 1 && runCommand.command.parameters[1].asynchronous){
+            ConsoleUtils.executeCommandAsync(exeCommandWithPath, dirPath, result,this.env );
```

Comment:
If a command is executed we have to clean everything up after the playbook was executed. So you have to pass the information to this.asyncProcesses.